exec/csnex/exclude.txt

```diff
@@ -240,7 +240,6 @@ string-test.neon
 strings.neon                                    # string__index
 struct-test.neon
 structure.neon
-sys-exit.neon                                   # sys module
```

Comment:
Can you also enable the `cmdline.neon` test?

Reply:
Yes.  It's enabled now.